content/docs/instrumenting/standard_exports.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Standard Exports
+sort_rank: 7
+---
+
+# Standard Exports
+
+This document covers what exports and collectors Prometheus client libraries
+should offer, with the aim of consistency across libraries, making the easy use
+cases easy and making it easier to provide tooling and consoles for common use
+cases.
+
+In addition, client libraries are ENCOURAGED to also offer whatever makes sense
+in terms of metrics for their language’s runtime (e.g. Garbage collection
+stats).
+
+These exports should have the prefix `process_`. If a language or runtime
+doesn't expose one of the variables it'd just not export it. All memory values
+in bytes, all times in unixtime/seconds.
+
+## Should
+
+| Metric name                     | Description                                            | Format  |
```

Comment:
Deffo! Then description can maybe be more about what it is, in case it's not clear (I had to google the different memory stuff for instance).

I copied the descriptions here from the Go implementation (first one I found, but java seems to have the same)